cluster-autoscaler/metrics/zalando_metrics.go

```diff
@@ -39,10 +50,18 @@ var (
 			Name:      "node_group_in_backoff",
 			Help:      "Whether or not a node group is in BackOff. 1 if it is, 0 otherwise.",
 		}, []string{nodeGroupNameLabel})
+
+	failedScaleDownCount = k8smetrics.NewCounterVec(
+		&k8smetrics.CounterOpts{
+			Namespace: caNamespace,
+			Name:      "failed_scale_down_total",
+			Help:      "Number of times scale-up operation has failed.",
```

Comment:
Should that say "scale-down" instead of "scale-up"?

Reply:
Fixed.